roles/horizon/tasks/main.yml

```diff
@@ -42,6 +42,21 @@
   notify:
     - reload apache
 
+- name: set horizon keystone policy fact
+  set_fact: policy_for_horizon=True
+  when: horizon.customize|default('False')|bool
+
+# Use custom policy to allow cloud_admin admin panel access
+- name: configure custom keystone policy
+  template:
+    src: "roles/keystone/templates/etc/keystone/policy.json"
+    dest: "/etc/openstack-dashboard/keystone_policy.json"
+    mode: 0644
```

Comment:
Why don't we want this policy file to have different permissions then policy files for the other services which currently are 0644?